### PR TITLE
LX-1571 need raw image artifact for GCP

### PR DESCRIPTION
**Pull request by grwilson**
ab-pre-push: http://selfservice.jenkins.delphix.com/job/devops-gate/job/master/job/appliance-build-orchestrator-pre-push/174/

### CODE REVIEW

File: scripts/run-live-build.sh [hunk 1]
```diff
@@ -123,7 +123,7 @@ fi
 # user (e.g. other software); this is most useful when multiple variants
 # are built via a single call to "make" (e.g. using the "all" target).
 #
-for ext in ova qcow2 upgrade.tar.gz vhdx vmdk; do
+for ext in ova qcow2 upgrade.tar.gz tar.gz vhdx vmdk; do
```
Comment (prakashsurya) [inline scripts/run-live-build.sh]: Can we use the extension “.gcp.tar.gz” (or equivalent) to better distinguish it from the upgrade image?

Comment (grwilson) [reply scripts/run-live-build.sh]: I will give this a shot and do a follow-on review if it works.

**Comment by prakashsurya** [comment]
@grwilson you need to fix the style/shell checks before it'll allow you to land this.

**Pull request merged**